kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8/kafkaproxyingresses.kroxylicious.io-v1.yml

```
@@ -69,11 +69,20 @@ spec:
                     clusterIP specifies that this ingress is for access within the same Kubernetes cluster using 
                     ClusterIP Kubernetes Services.
                   required: [ "protocol" ]
+                  x-kubernetes-validations:
+                    - message: "nodeIdentifiedBy mechanism 'serverNameIndication' is incompatible with TCP protocol"
```

More helpful to say what's required, than what's prohibited:
```suggestion
                    - message: "nodeIdentifiedBy mechanism 'serverNameIndication' requires protocol=TLS"
```